src/utils/source-map.js

```diff
@@ -34,6 +36,10 @@ function shouldSourceMap(): boolean {
   return prefs.clientSourceMapsEnabled;
 }
 
+function hasMappedSource(source: Source): boolean {
+  return !isOriginalId(source.id);
```

Comment:
hmm, two things:

1. I think this should be `source.get("id")` -> i saw an error in the console
2. I think we first need to check if it's original or generated and then check for the other id... right now we assume its generated